owl/SOMA-OBJ.owl

```diff
@@ -1682,6 +1704,28 @@ In robotics, it is not uncommon to encounter shapes that are flipped compared to
     </owl:Class>
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#Storage -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#Storage">
```

Comment:
You can just use Containment

Reply:
yes but it is far more than a Containment. For example; a spoon contains a liquid but it can not store it. Hence a need for a new dispositional concept. :)

Reply:
Yes, as Mihai mentioned, there needs to be a distinction between containment and storage